<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task 2 Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnEnterParagraph">
      <property name="text">
       <string>Enter and Save Paragraph</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReadParagraph">
      <property name="text">
       <string>Read and Display Paragraph</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEnterArray">
      <property name="text">
       <string>Enter and Save Array</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="minimumHeight">
       <number>200</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
